lib/katalyst/basic/auth/middleware.rb

```diff
@@ -44,11 +34,7 @@ def password
         end
 
         def default_password
-          Digest::MD5.hexdigest(default_username)
-        end
-
-        def enabled?
-          self.class.enabled?
+          Digest::SHA256.hexdigest(default_username)[0..15]
```

Comment:
This is pretty insecure because it's derived from an easily guessable value. Adding a salt would help, consider using `SECRET_KEY_BASE`

Reply:
good idea, I've added a salt (see Config class)